mavros/include/mavros/setpoint_mixin.h

```diff
@@ -162,8 +166,10 @@ class SetAttitudeTargetMixin {
 template <class D>
 class TF2ListenerMixin {
 public:
-	std::thread tf_thread;
-	std::string tf_thd_name;
+	std::thread tf_thread_1;
+	std::thread tf_thread_2;
+	std::string tf_thd_name_1;
+	std::string tf_thd_name_2;
```

Comment:
Each plugin instance (object) have their `std::thread` object.
It is not **shared** between plugins, only inside class instance.